ahs week6 assignment submission

AaronShtilerman opened the pull request.

Aaron-Shtilerman committed:
ahs week6 assignment submission

geena-panzitta commented:
Hi Aaron,
I thought you did a great job on the survey! I ran it multiple times trying to find any bugs and everything ran smoothly! I’ve included my comments below:

For the state question (row 20 on the google sheets), you could use a drop down/multiple choice format pulling from a list of all the states.

For the specific targeting (row 39), it might be good to make this a “select all that apply” kind of question.

When it asks for the number of classes you’re taking (row 23), it lets you input 0, which doesn’t seem like it would be possible if you were a current student.

Similarly, for the number of hours the class meets (row 30), it lets you input 0 which doesn’t seem likely.

I’m not sure if this is just a SurveyCTO thing, but when you go back and change the number of classes (row 23) to something smaller than you originally had (such as changing it from 3 to 1), it still asks for the original number of classes/still repeats the same as before (though if you change it to something bigger, it adds more repeats).

I saw on the google sheets where you had a follow up question (row 43), but I wasn’t able to prompt the question no matter if I selected “math” or “accounting”.

geena-panzitta reviewed (approved):
Looks great!

AaronShtilerman commented:
Thanks for reviewing my submission!

For the state question, I wanted to be inclusive of states outside of the US. For a legitimate survey, I would use a dataset of all the states, but in this case I just left it as open-ended.

Specific targeting was initially a select multiple, I changed it to a select one because this hypothetical survey wouldn't care how many classes they are taking in the targeted fields, just if they were taking any classes.

For number of classes, makes sense, changed it to a minimum of 1.

For class hours, I let 0 be an option because completely asynchronous classes could exist. 

I did actually notice that issue while running debugging of my own- I'm not sure if there is a good way around that- it's why I add a confirmation message before I start the repeat loop, to enable second chances before starting. 

The last one, I was stumped for a bit- but I realized that I added that question and forgot to revise the survey- the survey should be updated now!
